From a12baac2c7f33a6c1ec037e88a30fe3c5f0dcd90 Mon Sep 17 00:00:00 2001
From: Tomasz Michalec <tomasz@marvell.com>
Date: Tue, 3 Sep 2019 16:15:23 +0200
Subject: [PATCH 0277/1921] octeontx2-af: Add T98 devid to PTP id table

This patch allows to attach Marvell PTP Driver to the PTP device
on t98.

Change-Id: I7dad35443c8ad96c6529f4dd2eb6c0bdf9753013
Signed-off-by: Tomasz Michalec <tomasz@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/15215
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/ptp.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
index 1bca8303d0eb..c85ee22b3d2b 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
@@ -17,6 +17,7 @@
 #define DRV_NAME	"Marvell PTP Driver"
 
 #define PCI_DEVID_OCTEONTX2_PTP		0xA00C
+#define PCI_SUBSYS_DEVID_OCTX2_98xx_PTP	0xB100
 #define PCI_SUBSYS_DEVID_OCTX2_96XX_PTP	0xB200
 #define PCI_SUBSYS_DEVID_OCTX2_95XX_PTP	0xB300
 #define PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP	0xB400
@@ -217,6 +218,9 @@ static void ptp_remove(struct pci_dev *pdev)
 }
 
 static const struct pci_device_id ptp_id_table[] = {
+	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
+			 PCI_VENDOR_ID_CAVIUM,
+			 PCI_SUBSYS_DEVID_OCTX2_98xx_PTP) },
 	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
 			 PCI_VENDOR_ID_CAVIUM,
 			 PCI_SUBSYS_DEVID_OCTX2_96XX_PTP) },
-- 
2.31.1

